applications/processor/twitter-trend-processor/src/test/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+		</encoder>
+	</appender>
+	<root level="INFO">
+		<appender-ref ref="STDOUT" />
+	</root>
+	<logger name="org.mockserver" level="WARN" />
+	<logger name="com.networknt.schema" level="WARN" />
+</configuration>
```

Comment:
New line in every single file in the project.

There has to be an option in the IDE to "add new line in the end on file save":
![image](https://user-images.githubusercontent.com/1071590/170512431-4bf18dac-d5e8-4163-872c-ffb547834f65.png)

Reply:
I updated and had to force push because I had to reset the commits in order for IDEA to update the line endings. Should be good to go.